public/app/features/explore/LiveTailButton.tsx

```diff
@@ -107,25 +96,23 @@ export function LiveTailButton(props: LiveTailButtonProps) {
   const { start, pause, resume, isLive, isPaused, stop, splitted } = props;
   const theme = useTheme();
   const styles = getStyles(theme);
+  const buttonVariant = isLive && !isPaused ? 'active' : 'default';
 
   const onClickMain = isLive ? (isPaused ? resume : pause) : start;
 
   return (
-    <>
+    <ButtonGroup>
       <Tooltip content={isLive ? <>Pause the live stream</> : <>Live stream your logs</>} placement="bottom">
-        <ResponsiveButton
-          splitted={splitted}
-          buttonClassName={classNames('btn navbar-button', styles.liveButton, {
-            [`btn--radius-right-0 explore-active-button ${styles.noRightBorderStyle}`]: isLive,
-            [styles.isLive]: isLive && !isPaused,
-            [styles.isPaused]: isLive && isPaused,
-          })}
+        <ToolbarButton
+          iconOnly={splitted}
+          variant={buttonVariant}
           icon={!isLive ? 'play' : 'pause'}
```

Comment:
This should be using `isPaused`. The confusion is that `isLive` means you are in live tailing mode, which then can be running or paused. This bug is here since 7.0 though.

Reply:
fixed